releasenotes/notes/Add-ability-to-pass-breakpoint-flag-to-Python-checks-3d289df93c6d609b.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Add the ability to pass down the ``set_breakpoint`` flag to Python checks to start
```

Comment:
This should mention that this is available in the `datadog-agent check` command. I also don't think we need to mention `set_breakpoint` which seems internal.